lib/posix/options/Kconfig.signal

```diff
@@ -28,13 +36,6 @@ config POSIX_SIGNAL_STRING_DESC
 	  Use full description for the strsignal API.
 	  Will use 256 bytes of ROM.
 
-config POSIX_LIMITS_RTSIG_MAX
-	int "_POSIX_RTSIG_MAX value in limits.h"
-	default 8
-	help
-	  Define the _POSIX_RTSIG_MAX value in limits.h.
-	  IEEE 1003.1 defines this to be 8.
-
```

Comment:
I think this was implementing `_POSIX_RTSIG_MAX`:

https://pubs.opengroup.org/onlinepubs/7908799/xsh/limits.h.html
```
RTSIG_MAX
Maximum number of realtime signals reserved for application use in this implementation. Minimum Acceptable Value: _POSIX_RTSIG_MAX
```
```
_POSIX_RTSIG_MAX
The number of realtime signal numbers reserved for application use. Value: 8
```

Reply:
This way uses one fewer Kconfig variable and puts the default minimum in Kconfig.